### PR TITLE
Test(web): Introduce SASS unit tests

### PR DESCRIPTION
**Pull request by crishpeen**
## Description

Let me introduce you to SASS unit tests!

They run using Jest and you can test anything using sass-test.

TODO:
- [x] Write contribution info
- [x] Write more tests

---

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md).
- [x] Follow the [PR Title/Commit Message Convention](https://github.com/lmc-eu/spirit-design-system/blob/main/CONTRIBUTING.md#commit-conventions).
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [x] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: packages/web/src/scss/tools/__tests__/_accessibility.test.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
+// stylelint-disable scss/at-mixin-argumentless-call-parentheses -- We need to allow this to make sass-true work
+@use 'true';
+@use '../accessibility';
+
+@include true.describe('hide-text mixin') {
+    @include true.it('outputs styles to visually hide text') {
```
Comment (literat) [inline packages/web/src/scss/tools/__tests__/_accessibility.test.scss:6]: Please, use `should` word to start the sentence of the test description like `it should outputs ...`

File: packages/web/src/scss/tools/__tests__/_accessibility.test.scss [hunk 2]
```diff
@@ -0,0 +1,82 @@
+// stylelint-disable scss/at-mixin-argumentless-call-parentheses -- We need to allow this to make sass-true work
+@use 'true';
```
Comment (literat) [inline packages/web/src/scss/tools/__tests__/_accessibility.test.scss:2]: Can be `true` used somehow globally? Like we are not writing `true.describe` but only `describe`...?

**Comment by literat** [comment]
👏 I am very happy to see this PR. It really helps me to understand what is going on in the SCSS mixins 😅 Thanks a lot ❤️

File: packages/web/src/scss/tools/__tests__/_accessibility.test.scss (hunk 1 above)
Comment (crishpeen) [reply packages/web/src/scss/tools/__tests__/_accessibility.test.scss:6]: added

File: packages/web/src/scss/tools/__tests__/_accessibility.test.scss (hunk 2 above)
Comment (crishpeen) [reply packages/web/src/scss/tools/__tests__/_accessibility.test.scss:2]: AFAIK not :)

File: packages/web/CONTRIBUTING.md [hunk 3]
```diff
@@ -126,6 +126,13 @@ Now use the reference from the theme in component styles:
 - `% yarn test` for test package (lint, format, unit testing, types)
 - `% yarn test:unit` for unit tests
 
+### SASS Unit Testing
+
+We use [sass-true][sass-true] for unit testing of our SASS components. The
+tests are located in `src/scss/components/<ComponentName>/__tests__` and are
+named `<ComponentName>.test.scss`. The tests are run as a part of the Jest
+testing suite. We aim to have 100% coverage of our mixins and functions.
```
Comment (literat) [inline packages/web/CONTRIBUTING.md:134]: > We aim to have 100% coverage of our mixins and functions.

I think it is obvious that we want to have a `100 %` (with space) coverage. But it is embarrassing if this sentence will be here and the coverage will not be there :-/

Comment (crishpeen) [reply packages/web/CONTRIBUTING.md:134]: :D ok, I will rewrite it to highlight we focus on testing mixins and functions :) and I won't mention any percentages :D

**Review by literat — APPROVED** [review]
👏

**Review by pavelklibani — APPROVED** [review]

**Pull request merged**